guides/Git.md

```diff
@@ -42,53 +49,55 @@ Write a [good commit message]. Example format:
 
     http://project.management-system.com/ticket/123
 
-If you've created more than one commit, use a rebase to squash them into cohesive commits with good messages:
+[good commit message]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
 
-    git rebase -i origin/dev
 
-Share your branch.
+## Request a Review
 
-    git push origin ${branch_name}
+If you've created more than one commit, use interactive rebase to squash them into cohesive commits with good messages:
 
-Submit a pull request, assign it to code reviewer, add ‘needs_review’ label. (**)
+    git rebase -i origin/master
 
-[good commit message]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
 
-## Review Code
+Share your branch.
 
-When someone else makes a pull request, you can review their changes.
-Make comments and ask questions directly on lines of code in the GitHub web interface.
+    git push origin ${branch_name}
 
-When finished, remove ‘needs_review’ label.
+Create a [pull request] on Github and have a team mate review it for you. They
+will make comments and ask questions via the web interface, or discuss it with you directly.
 
-If you’re satisfied with the changes, or there are only minor styling issues, add ‘ship_it’ label. 
-If pull request contains bugs or several issues, add ‘pending_changes’ label.
-If you think pull request changes are not needed at all, add ‘do_we_need_this?’ label.
+[pull request]: https://help.github.com/articles/using-pull-requests
```

Comment:
I decided no to include this article, as I thought it doesn't bring that much value. I think the concept of pull requests is easy to understand. Maybe we can just add one sentence here in the guide?